src/ucp/core/ucp_worker.c

```diff
@@ -33,6 +33,24 @@ static ucs_stats_class_t ucp_worker_stats_class = {
         [UCP_WORKER_STAT_TAG_RX_RNDV_UNEXP]        = "rx_rndv_rts_unexp"
     }
 };
+
+static ucs_stats_class_t ucp_worker_tm_offload_stats_class = {
+    .name           = "tag_offload",
+    .num_counters   = UCP_WORKER_STAT_TAG_OFFLOAD_LAST,
+    .counter_names  = {
+        [UCP_WORKER_STAT_TAG_OFFLOAD_POSTED]           = "posted",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_CANCELED]         = "canceled",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_BLOCK_TAG_EXCEED] = "block_tag_exceed",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_BLOCK_NON_CONTIG] = "block_non_contig",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_BLOCK_WILDCARD]   = "block_wildcard",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_BLOCK_SW_PEND]    = "block_sw_pend",
+        [UCP_WORKER_STAT_TAG_OFFLOAD_RX_COMPLETED]     = "rx_completed",
```

Comment:
maybe call it "matched" (and UCP_WORKER_STAT_TAG_OFFLOAD_MATCHED)?
i would also move it after "posted" since these are the 2 most interesting counters - how much was posted, how much was matched